optionals/compat_rhs_usf3/CfgAmmo.hpp

```diff
@@ -40,7 +40,19 @@ class CfgAmmo {
         ACE_barrelLengths[] = {508.0, 609.6, 660.4};
     };
     class B_556x45_Ball;
-    class rhs_ammo_556x45_M855A1_Ball: B_556x45_Ball { // B_556x45_Ball (ballistics/CfgAmmo.hpp) 
```

Comment:
Sometimes I wish diffing algo's were a little better.

Reply:
You changed the line break.

Reply:
It's supposed to be LF. Either it was CRLF or you changed it to it, either way .gitattributes apparently doesn't catch all configurations. :/

Reply:
Everything is CRLF in my file \(o_o)/

Reply:
btw .gitattributes doesn't say that it will convert to anything.
I see editorConfig says LF.. But the .editorconfig is using CRLF itself..
I guess that's just autocrlf doing stuff...

Reply:
Yes that's `autocrlf`, Git will automatically convert to LF as by `.gitattributes`, you won't see those messages if you are using `autocrlf` because that's the point of `autocrlf`.

And yes, it obviously does say: `* text=auto`. The whole point is that it doesn't matter what your local files are using, they should always be LF on the server (GitHub), although we had an occasion in the past that it somehow slipped through, but it seemed to be because of editing directly on GitHub.